File: backend/src/main/java/com/Fijalkowskim/SpaceFarmManagmentSystem/requestmodels/HarvestRequest.java
package com.Fijalkowskim.SpaceFarmManagmentSystem.requestmodels;

import lombok.Data;

import java.util.Date;

@Data
public class HarvestRequest {

    private Date harvestDate;

    private boolean successfulHarvest;

    private String comment;

    private long cultivationId;
}

File: frontend/src/models/requestmodels/HarvestRequest.js
export class HarvestRequest {
  constructor(harvestDate, successfulHarvest, comment, cultivationId) {
    this.harvestDate = harvestDate;
    this.successfulHarvest = successfulHarvest;
    this.comment = comment;
    this.cultivationId = cultivationId;
  }
}
